source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -0,0 +1,465 @@
+=================
+Server Monitoring
+=================
+
+:Spec: 1580
+:Title: Server Monitoring
+:Author: Shane Harvey
+:Advisors: Matt broadstone
+:Status: Accepted
+:Type: Standards
+:Version: 2.15
```

Comment:
I was considering it the same spec. So 2.16 is correct. Hopefully we remember to keep it in sync.

Reply:
There is almost no chance we will keep the version numbers in sync. 

The price we have to pay for separate documents is separate version numbers for each document.  See https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst, for example, which is at 1.1.1.

So let's bite the bullet and call this 1.0.

Reply:
Done.

Reply:
@mbroadst let us know if you're ok with this.

If we want a single version number, we need a single place to put that version number, and I can't think of a place to put it that we're going to remember to update.

Reply:
@jyemin I don't think I'm okay with that, I think it will ultimately be very confusing. I think of this as just a "chapter" in the SDAM spec, not a spec in its own right. The version number is probably most important for the tests themselves, not so much the prose of the specifications themselves. It seems we have a number of choices:
- allow different files in this folder to have different versions
- move everything into a single file under a single version
- sync version numbers across all files that comprise the "SDAM spec"
- split the specs into their own folders and maintain their separate versions 

The last option I think illuminates the problem with multiple versions in the same "package". Currently, the tests for monitoring-monitoring are mixed in with the regular sdam tests, but those specs are at different versions. How does one reconcile that?

I'm a fan of option number #2 here, use a common version for every file. It was an arbitrary choice that a different version was chosen for monitoring, and I think there is a case here that we don't currently use any tooling that makes the version meaningful.

Reply:
These are the four options I see for right now:
- ```:Version: Same as the `main SDAM spec`_.```
- Delete the version field.
- ```:Version: 2.16```
- ```:Version: 1.0```

At this point I prefer one of the first two options.

Reply:
Either of those options SGTM

Reply:
Updated to ```:Version: Same as the `Server Discovery And Monitoring`_ spec```.

Reply:
This satisfies my concern that the version numbers remain in sync at least.  We will have to be vigilant going forward about updating the version number in the main document when any of the "chapters" change.  We'll see how it goes.

Given this decision, please also apply it to https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst.

Reply:
Done.